main.c

```diff
@@ -782,22 +782,27 @@ static struct option long_options[] = {
 
 int main(int argc, char **argv)
 {
-	int ret;
+	darray(struct tcmulib_handler) handlers = darray_new();
+	struct tcmulib_context *tcmulib_context;
+	struct tcmur_handler **tmp_r_handler;
 	GMainLoop *loop;
 	GIOChannel *libtcmu_gio;
 	guint reg_id;
-	int c;
-	struct tcmulib_context *tcmulib_context;
-	darray(struct tcmulib_handler) handlers = darray_new();
-	struct tcmur_handler **tmp_r_handler;
+	int ret;
 
 	tcmu_cfg = tcmu_config_new();
 	if (!tcmu_cfg)
 		exit(1);
-	tcmu_load_config(tcmu_cfg, NULL);
+	ret = tcmu_load_config(tcmu_cfg, NULL);
+	if (ret == -1)
+		goto err_out;
+
+	/* set the log level */
+	tcmu_set_log_level(tcmu_cfg->log_level);
```

Comment:
Sorry, This just added one comment before the dyn reload PR(And this PR has removed this).
To resolve the confliction I introduced it back again.

Will remove it right now.

Thanks,